lib/layouts/settings/theme_panel.dart

```diff
@@ -166,7 +166,7 @@ class ThemePanel extends GetView<ThemePanelController> {
                         saveSettings();
                         controller.update();
                       },
-                      options: Skins.values.toList(),
+                      options: Skins.values.where((item) => item != Skins.Samsung).toList(),
```

Comment:
didn’t you say this could cause an error for people who have it enabled already or no?

Reply:
yes, because it'll try to find the selected item as `Skins.Samsung` but that doesn't appear in the options list.

Reply:
Should we fix that then? or should we add a case where if it's not in the list, default it to something

Reply:
Ehh idk since its a change from alphas, not stable. But I might be able to cook up something later today around 5 ish

Reply:
Cooked something up rq so we should be good here